pkg/algorithm/hived_algorithm.go

```diff
@@ -775,12 +836,37 @@ func (h *HivedAlgorithm) validateSchedulingRequest(sr schedulingRequest, pod *co
 // processSchedulingRequest feeds a request to a VC scheduler or the opportunistic scheduler depending on its priority.
 func (h *HivedAlgorithm) processSchedulingRequest(
 	sr schedulingRequest,
-	suggestedNodes common.Set) (groupPhysicalPlacement, groupVirtualPlacement) {
+	suggestedNodes common.Set) (
+	physicalPlacement groupPhysicalPlacement,
+	virtualPlacement groupVirtualPlacement,
+	nodesNotInSuggested common.Set) {
 
-	if sr.priority >= minGuaranteedPriority {
-		return h.scheduleGuaranteedAffinityGroup(sr, suggestedNodes)
+	str := fmt.Sprintf("chain %v", sr.chain)
+	if sr.reservationId != "" {
+		str = fmt.Sprintf("reservation %v", sr.reservationId)
+	}
+	klog.Infof("Processing scheduling request: %v, GPU numbers %v, priority %v",
+		str, common.ToJson(sr.affinityGroupPodNums), sr.priority)
```

Comment:
Do not show this log if you just return at line 857:

return nil, nil, nodesNotInSuggested